rakefile.rb

```diff
@@ -56,14 +58,15 @@ def write_docker_image_info(release_directory)
   end
   `echo #{build_num} > #{CYTOVALE_PROJECT}/.build_num`
   `echo 'CYM_BSP_BUILD_NUMBER = "#{build_num}"' > #{CYTOVALE_PROJECT}/.extras.conf`
+  `echo 'CYM_BSP_VERSION = "#{CYM_BSP_VERSION}"' >> #{CYTOVALE_PROJECT}/.extras.conf`
   bitbake(CYTOVALE_PROJECT, "--postread=.extras.conf #{CYTOVALE_IMAGE}")
 end
 
 desc "Build a release package for the Cytovale project"
 task :cytovale_release => [:cytovale_image] do
   build_time = Time.now.strftime("%Y-%m-%d-%H-%M-%S")
   build_date_str = Time.now.getutc.strftime("%Y%m%d")
-  release_directory = File.join(RELEASE_FOLDER, CYTOVALE_PROJECT, CYTOVALE_IMAGE, build_time)
+  release_directory = File.join(RELEASE_FOLDER, CYTOVALE_PROJECT, CYTOVALE_IMAGE, CYM_BSP_VERSION, build_time)
   deploy_dir = File.join(PROJECT_ROOT, "deploy", "images", "apalis-imx6")
```

Comment:
Since the zip file has the name of the release in it, this may not be necessary. If you change the release directory, you'll have to change the build instructions in the cytovale-sw repo.

Reply:
This is nice to have when you've built several BSPs.  I'll add it to the build instructions.